CONTRIBUTING.md

```diff
@@ -114,6 +114,16 @@ cd api-bench
 npm run coai
 ```
 
+#### Running Tests Locally
+
+To see a list of available CI tasks and their corresponding commands to run them locally, you can run `npm run local` in the source folder. This is particularly useful for testing specific CI tasks before committing your changes.
+
+```bash
+git clone https://github.com/idrinth/api-bench
+cd api-bench
+npm run local
```

Comment:
I doubt anyone would clone the repo just to get a list of tests, please remove the code here